experiments/aws_az/aws_az_chaos/experiment/aws_az_chaos.py

```diff
@@ -0,0 +1,132 @@
+import pkg.types.types  as types
+import pkg.aws_az.types.types as experimentDetails
+import pkg.aws_az.environment.environment as experimentEnv
+import pkg.events.events as events
+import logging
+import pkg.status.application as application
+import chaosLib.litmus.aws_az_chaos.lib.aws_az_chaos as litmusLIB
+import pkg.result.chaosresult as chaosResults
+from pkg.utils import client
+import pkg.utils.common.common as common
+import pkg.aws_status.status as awsStatus
+
+# Experiment contains steps to inject chaos
```

Comment:
A piece of general advice for the comments which are to serve as the description of any function: We should put them in doctrings within the function in the very first line rather than using general comments as that will count as proper documentation for the function via VSCode IntelliSense. Otherwise, the IntelliSense won't detect single-line comments as function description documentation.

Reply:
We will be having another PR for that